### PR TITLE
Adds semver change to PR template

### PR DESCRIPTION
**Pull request by brendankowitz**
## Description
Adds the proposed Semver change to the PR description so we can review with the PR change content

## FHIR Team Checklist
- [x] **Update the title** of the PR to be succinct and less than 50 characters
- [x] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [x] Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
- [x] Tag the PR with **Azure API for FHIR** if this will release to the managed service
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Skip (documentation)

### CODE REVIEW

**Review by LTA-Thinking — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -12,4 +12,7 @@ Describe how this change was tested.
 - [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
 - [ ] Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
 - [ ] Tag the PR with **Azure API for FHIR** if this will release to the managed service
-- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)
\ No newline at end of file
+- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)
+
+### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
```
Comment (Ivanidzo4ka) [inline .github/pull_request_template.md:17]: `\([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md)\)`

**Review by Ivanidzo4ka — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -12,4 +12,7 @@ Describe how this change was tested.
 - [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
 - [ ] Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
```
Comment (rbans96) [inline .github/pull_request_template.md:13]: This is unrelated to the semver change. Just a thought before I forget, probably we can include 'Dependencies' too.
'Tag the PR with the type of update: **Bug**, **Enhancement**, **Dependencies** or **New-Feature**'

Comment (brendankowitz) [reply .github/pull_request_template.md:13]: Yep sure, I don't think we should be limited to these tags
